### ISSUE TITLE
Add multiple device group to user group at once

### ISSUE THREAD

**Issue by Tomatoschewps** [Ylianst/MeshCentral#4197]
I want to create a group for our commercial team and add this group to all device group with limited rights.

Is it possible to add an option to "add multiple groups" to the device group section of users group page ?
![image](https://user-images.githubusercontent.com/10291238/176637216-7e606031-9a74-49bd-a9b6-f8a78ad68937.png)

If it allow us to select multiple device groups at once and add them with the same rights it will be perfect.

With this it will also possible to correct missing device groups in user group regularly.

Thanks,
D.

**Comment by si458** [comment]
you sadly have to add each group one by one at the moment,
but this is indeed a feature request that maybe @Ylianst could look into
as im not sure how you would display the add device group window showing each group and permissions on 1 screen while you add them?

**Comment by Tomatoschewps** [comment]
Yes, and I have undreds of groups that I want to add with specific rights so for now it's 6 click to add one group ^^

Same window as "add a group" but instead of a dropdown list, a multiselectable list (with ctrl and shift to select one by one or multiple contiguous) ?

**Comment by VINISHVV** [comment]
> Yes, and I have undreds of groups that I want to add with specific rights so for now it's 6 click to add one group ^^
> 
> Same window as "add a group" but instead of a dropdown list, a multiselectable list (with ctrl and shift to select one by one or multiple contiguous) ?

Try user group once.

**Comment by Tomatoschewps** [comment]
Hi @VINISHVV, I don't understand your post, sorry ...

**Comment by Tomatoschewps** [comment]
Hi @Ylianst,
With the last update of meshcentral and ldap group sync can you look at this ?
We now have synced group from our ldap but add those ldap users group to meshcentral device group will take us loooong time if we can't do it at once.
Thanks.
D.
